hooks/notification/.helm-docs.gotmpl

```diff
@@ -252,7 +252,11 @@ notificationChannels:
     endPoint: "someone@somewhere.xyz"
 env:
   - name: SMTP_CONFIG
-    value: "smtp://user:pass@smtp.domain.tld/"
+    # you can create the secret via: kubectl create secret generic email-credentials-token --from-literal="smtp-config=smtp://user:pass@smtp.domain.tld/"
+    valueFrom:
+      secretKeyRef:
+        name: email-credentials
```

Comment:
This seems inconsistent: `email-credentials-token` (in the kubectl create) vs. `email-credentials` (in the secretKeyRef) - can you double-check this?